x-pack/plugins/ml/public/application/jobs/jobs_list/components/delete_job_modal/delete_job_modal.tsx

```diff
@@ -104,17 +109,21 @@ export const DeleteJobModal: FC<Props> = ({ setShowFunction, unsetShowFunction,
                 </div>
               </div>
             ) : (
-              <EuiText>
-                <FormattedMessage
-                  id="xpack.ml.jobsList.deleteJobModal.deleteMultipleJobsDescription"
-                  defaultMessage="Deleting {jobsCount, plural, one {a job} other {multiple jobs}} can be time consuming.
+              <>
+                {hasManagedJob ? (
+                  <EuiText>
+                    <FormattedMessage
+                      id="xpack.ml.jobsList.deleteJobModal.deleteMultipleJobsDescription"
+                      defaultMessage="{jobsCount, plural, one {This job was} other {At least one of these jobs was}} deployed as part of a module; deleting {jobsCount, plural, one {it} other {them}} might impact other parts of the product.
```

Comment:
As above, we should drop the term 'module' here, and say something like `This pre-configured job was provided by Elastic...`

Reply:
Updated here https://github.com/elastic/kibana/pull/122305/commits/427be5b2ff13c2453e9b03611547f978baa19cbc